rhizome/inference_endpoint/lib/replica_setup.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require_relative "../../common/lib/util"
+
+require "fileutils"
+require "netaddr"
+require "json"
+require "openssl"
+require "base64"
+require "uri"
+
+class ReplicaSetup
+  def prep(inference_engine:, inference_engine_params:, model:, replica_ubid:, ssl_crt_path:, ssl_key_path:, use_self_signed_cert:, gateway_port:)
+    engine_start_cmd = engine_start_command(inference_engine: inference_engine, inference_engine_params: inference_engine_params, model: model)
+    write_config_files(replica_ubid, ssl_crt_path, ssl_key_path, gateway_port)
+    install_systemd_units(engine_start_cmd, use_self_signed_cert)
+    start_systemd_units
+  end
+
+  def start_systemd_units
+    r "systemctl start lb-cert-download.timer"
+    r "systemctl start inference-engine.service"
+  end
+
+  def inference_gateway_service
+    "/etc/systemd/system/inference-gateway.service"
+  end
+
+  def write(path, s)
+    s += "\n" unless s.end_with?("\n")
+    File.write(path, s)
```

Comment:
I think `puts` has these semantics "for free"

Reply:
thanks. changed to `puts`.